### PR TITLE
add activity tracker

### PR DESCRIPTION
**Pull request by MikeBirdTech**
ai log doing work
ai log
ai log tell me how long i was doing work
ai log category list
ai log category rename "Coding" "Programming"
ai log category merge "Dev" "Programming"
ai log category show "Programming"

It's pretty cool!

### CODE REVIEW

File: pyproject.toml
Comment (MikeBirdTech) [inline pyproject.toml:1]: This is probably wrong but i had to do it to run my script. I don't know why the cal imports mattered when i was running log

File: src/tool_use/scripts/activity_tracker.py [hunk 1]
```diff
@@ -0,0 +1,609 @@
+from pathlib import Path
+import sqlite3
+import time
+import datetime
+from typing import Optional, Tuple, List
+from rich.console import Console
+from rich.table import Table
+from ..utils.ai_service import AIService
+
+HELP_TEXT = """Usage: ai log [command] [args]
```
Comment (MikeBirdTech) [inline src/tool_use/scripts/activity_tracker.py:10]: Help text might not belong here, but I thought keeping it close to the code it explains keeps is easy to update for now

File: src/tool_use/scripts/activity_tracker.py [hunk 2]
```diff
@@ -0,0 +1,609 @@
...
+  help              Show this help message
+
+Examples:
+  ai log category list
+  ai log category rename "Coding" "Programming"
+  ai log category merge "Dev" "Programming"
+  ai log category show "Programming"
+"""
+
+
```
Comment (MikeBirdTech) [inline src/tool_use/scripts/activity_tracker.py:44]: directory/config stuff should probably be extracted to a utility at some point, not sure if it's worthwhile to do now

**Comment by MikeBirdTech** [comment]
@mentatbot please take a look

File: pyproject.toml
Comment (tyfiero) [reply pyproject.toml:1]: weird! works for now, ill look into it

**Review by tyfiero — APPROVED** [review]

**Pull request merged**